src/App.js

```diff
@@ -1,21 +1,16 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import  CenteredGrid from './cardListView/CardListView'
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+        <CenteredGrid/>
```

Comment:
do not change this file.
eventually this file will combine all but for now this is part of the home screen

Reply:
why?
you need to change app.js if you want to change your app...

Reply:
Ok, you are right
when the other component will be implemented we will add them too.